hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -5516,6 +5522,10 @@ public boolean hasPathCapability(final Path path, final String capability)
     case FIPS_ENDPOINT:
       return fipsEnabled;
 
+    // is S3 Access Grants enabled
+    case AWS_S3_ACCESS_GRANTS_ENABLED:
```

Comment:
our bucket-info command has a list of capabilities to probe for -add this to the list